fulp_modules/main_features/species/beefmen/mob/beefman.dm

```diff
@@ -722,3 +723,34 @@
 /datum/sprite_accessory/beef/mouth/smile2
 	name = "Smile2"
 	icon_state = "smile2"
+
+/obj/item/organ/tongue/beefman_tongue
+	name = "meaty sausage tongue"
+	desc = "A meaty and thick muscle typically found in Beefmen"
```

Comment:
```suggestion
	desc = "A meaty and thick muscle typically found in Beefmen."
```

last one i swear

Reply:
OK man but don’t think I don’t KNOW what’s going on here. CLEARLY ur a tyrant and U just don’t want Beefmen to have freedom of SPEECH. guess what????? I’m GOING to KEEP fighting U and this day WILL go dowN in history as The DAY I deFeated John Willard in the Name OF BEEFMEN civil LIBERTIES

Reply:
i have your ip address

Reply:
hey man I was JOKING calm down OK? Listen when I GO home I’ll add ur GODDAMNED full stop